cubecrit/templates/manufacturer.html

```diff
@@ -0,0 +1,75 @@
+{% extends "base.html" %}
+{% block title %}
+    {{ manufacturer.display_name }}
+{% endblock title %}
+{% block content %}
+    <h1 class="display-2 text-center">{{ manufacturer.display_name }}</h1>
+    <blockquote class="blockquote text-bg-light p-3 w-75 m-auto mb-3">
```

Comment:
Can you change `text-bg-light` to `bg-body-tertiary`? It works the same in light mode but looks better when using dark mode.

`text-bg-light`:
<img width="1367" alt="Screenshot 2024-10-13 at 8 34 37 PM" src="https://github.com/user-attachments/assets/b42da4e4-bb83-4538-a882-6c38e7f95d67">

`bg-body-tertiary`:
<img width="1373" alt="Screenshot 2024-10-13 at 8 35 39 PM" src="https://github.com/user-attachments/assets/e2793dcf-f7a7-4021-9dd1-119c7258b7c1">